docs/hub/security.md

```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
+
+Scanning is triggered at each commit or when you visit a repository page.
```

Comment:
Do we trigger the scanning every time any user goes to a repo?

Reply:
Yes, if it hasn't been scanned already (for a given revision)